docs/sources/release-notes/v2.3.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Grafana Mimir version 2.3 release notes"
+menuTitle: "V2.3 release notes"
+description: "Release notes for Grafana Mimir version 2.3"
+weight: 300
+---
+
+# Grafana Mimir version 2.3 release notes
+
+Grafana Labs is excited to announce version 2.3 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.2, there is upgrade-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Ingest metrics in OpenTelemetry format**:
+  This release of Grafana Mimir introduces experimental support for ingesting metrics from the [OpenTelemetry Collector's](https://opentelemetry.io/docs/collector/) [`otlphttp` exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlphttpexporter/README.md). This adds a second ingestion option for users of the OTel Collector; Mimir was already compatible with the [`prometheusremotewrite` exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusremotewriteexporter). For more information, please see [Configure OTel Collector]({{< relref "../operators-guide/configure/configure-otel-collector.md">}}).
+
+- **Increased instant query performance**:
+  As an experimental feature, instant queries in Grafana Mimir now will get the same benefits of query sharding we see for range queries. When enabled with `-query-frontend.split-instant-queries-by-interval`, splitting instant queries by time will contribute in some cases to a speed up in instant query response time.
```

Comment:
```suggestion
Grafana Mimir now supports splitting instant queries by time. This allows it to better parallelize execution of instant queries and therefore return results faster. At present, splitting is only supported for a subset of instant queries, which means not all instant queries will see a speedup. This feature is being released as experimental and is disabled by default. It can be enabled by setting `-query-frontend.split-instant-queries-by-interval`.
```